src/EgressClient.ts

```diff
@@ -50,6 +50,7 @@ export default class EgressClient {
    * @param audioOnly record audio only
    * @param videoOnly record video only
    * @param customBaseUrl custom template url
+   * @param disableManifest disable upload of json manifest file
    */
   async startRoomCompositeEgress(
```

Comment:
I'm starting to think we should move these options into an object, instead of more arguments. (it's getting a bit harder to keep track of).

maybe something like
```
RoomCompositeOptions {
  encoding: EncodingOptions
  audioOnly?: boolean
  ...
}
```

We could define multiple interfaces like [here](https://github.com/livekit/client-sdk-js/blob/main/src/room/track/Track.ts#L143), for backwards compatibility.